gcp/appengine/deploy.sh

```diff
@@ -3,13 +3,16 @@
 dir=$(dirname "$0")
 
 if [ $# -lt 1 ]; then
```

Comment:
```suggestion
if [[ $# -lt 2 ]]; then
```
I think we're going to be mandating two arguments now, right? Also adhere to https://google.github.io/styleguide/shellguide.html#test----and---